src/tests/suite/daos_obj.c

```diff
@@ -660,6 +693,29 @@ io_overwrite_large(void **state, daos_obj_id_t oid)
 		rec_idx += rx_nr; /* next index for insert/lookup */
 		/* Increment next overwrite size by 1 record extent */
 		rx_nr++;
+
+		/*Verify the SCM/NVMe Pool Free size based on tranfer size*/
+		pool_storage_info(state, &pinfo);
```

Comment:
do we need to check the return code here? or assert it's 0 ?

Reply:
Added